AndroidTVGateway/agent/app/src/main/java/org/wso2/androidtv/agent/siddhiSinks/EdgeGatewaySink.java

```diff
@@ -109,29 +140,29 @@ public void publish(Object o, DynamicOptions dynamicOptions) throws ConnectionUn
             try {
```

Comment:
Seems we have the same pattern for try{}catches{}. Use a single private method to do the validation.